### PR TITLE
feature: Use PropertyChanged.Fody

### PR DESCRIPTION
**Pull request by worldbeater**
There were some issues with <a href="https://github.com/egramtel/egram.tel/issues/14">ReactiveUI.Fody</a> so we decided to use <a href="https://github.com/fody/propertychanged">PropertyChanged.Fody</a>. With this package the project compiles successfully both on Windows and on Linux (as far as I tested). But we should remember one rule when using this package:

* We should not implement `INotifyPropertyChanged` interface by hands or inherit from `ReactiveObject`, otherwise there is a potential for an unexpected behavior. Instead, always use `[AddINotifyPropertyChanged]` attribute on each class that could be bound to the XAML UI.

### CODE REVIEW

File: Tel.Egram.Components.Application/ApplicationContext.cs [hunk 1]
```diff
@@ -11,56 +11,35 @@
...
-            _authenticator.ObserveState()
-                .ObserveOn(AvaloniaScheduler.Instance)
+            _authenticator
+                .ObserveState()
+                .ObserveOn(RxApp.MainThreadScheduler)
                 .Subscribe(state =>
-                {
+                {   
+                    Console.WriteLine("State update received.");
+                    Console.WriteLine(state.DataType);
```
Comment (x2bool) [inline Tel.Egram.Components.Application/ApplicationContext.cs]: Did you forget to remove debug prints?

File: Tel.Egram/Program.cs [hunk 2]
```diff
@@ -7,18 +7,19 @@
 
 namespace Tel.Egram
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
+            Console.WriteLine("Hello world");
```
Comment (x2bool) [inline Tel.Egram/Program.cs]: Did you forget to remove debug prints?

File: Tel.Egram.Components.Application/ApplicationContext.cs (hunk 1 above)
Comment (worldbeater) [reply Tel.Egram.Components.Application/ApplicationContext.cs]: Oops

**Comment by x2bool** [comment]
Thank you!

**Pull request merged**